### DISCUSSION TITLE
Missing Plugin endpoint

### DISCUSSION THREAD

**Discussion by Ricci1518** [danny-avila/LibreChat#1827] [Troubleshooting] [answered]
### What happened?

I installed 6.10 via docker (as described in the docs) using the .env.example (only adding keys for openai, dalle and ). Everything starts up. But I don't get to choose plugins in the drowdown (only OpenAI and Assistant)

### Steps to Reproduce

1. Install 
2. Copy .env and add API-keys
3. docker-compose up 
4. openAI endpoint working but no plugins

### Relevant log output

```shell
I did't find anything related
```

### Screenshots

![Bildschirmfoto vom 2024-02-17 22-15-54](https://github.com/danny-avila/LibreChat/assets/77723401/129cca79-cc22-4733-9816-c0830cf1bc05)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment, marked as answer]
Looks like you might have edited the `ENDPOINTS` var in the `.env` ? 
If you edited it, make sure it contains `gptPlugins`

example:
```sh
ENDPOINTS=openAI,assistants,gptPlugins
```

**Reply by Ricci1518** [reply]
Thank you for your quick reply.
You are right. I must have edited i, the day before and missed it yesterday.
